test/python/circuit/library/test_nlocal.py

```diff
@@ -1181,37 +1190,35 @@ def test_swaprz_circuit(self):
         parameters = ParameterVector("theta", num_qubits * (reps + 1) + reps * (num_qubits - 1))
         param_iter = iter(parameters)
 
-        #      ┌──────────┐┌────────────┐┌────────────┐ ┌──────────┐               »
-        # q_0: ┤ Rz(θ[0]) ├┤0           ├┤0           ├─┤ Rz(θ[5]) ├───────────────»
-        #      ├──────────┤│  Rxx(θ[3]) ││  Ryy(θ[3]) │┌┴──────────┴┐┌────────────┐»
-        # q_1: ┤ Rz(θ[1]) ├┤1           ├┤1           ├┤0           ├┤0           ├»
-        #      ├──────────┤└────────────┘└────────────┘│  Rxx(θ[4]) ││  Ryy(θ[4]) │»
-        # q_2: ┤ Rz(θ[2]) ├────────────────────────────┤1           ├┤1           ├»
-        #      └──────────┘                            └────────────┘└────────────┘»
-        # «                 ┌────────────┐┌────────────┐┌───────────┐               »
-        # «q_0: ────────────┤0           ├┤0           ├┤ Rz(θ[10]) ├───────────────»
-        # «     ┌──────────┐│  Rxx(θ[8]) ││  Ryy(θ[8]) │├───────────┴┐┌────────────┐»
-        # «q_1: ┤ Rz(θ[6]) ├┤1           ├┤1           ├┤0           ├┤0           ├»
-        # «     ├──────────┤└────────────┘└────────────┘│  Rxx(θ[9]) ││  Ryy(θ[9]) │»
-        # «q_2: ┤ Rz(θ[7]) ├────────────────────────────┤1           ├┤1           ├»
-        # «     └──────────┘                            └────────────┘└────────────┘»
+        #     ┌──────────────┐┌────────────────────────┐     ┌──────────────┐     »
```

Comment:
The spacing (and in the other place with a drawing) are a bit off 🙂

Reply:
also - why "theta" and not the symbol?